.github/CODEOWNERS

```diff
@@ -26,24 +26,28 @@
 # Codeowners ownership (for adding/changing/removing code owners).
 .github/CODEOWNERS @oppia/owners
 
+# TODO(#3164): Restore code ownership to @BenHenning after 2021-05-24.
 # Gradle build files.
-*.gradle @BenHenning
-gradle.properties @BenHenning
-gradlew @BenHenning
-gradlew.bat @BenHenning
-/gradle/ @BenHenning
+*.gradle @rt4914
```

Comment:
I think this file might have been merged wrong. Could you copy the latest version of CODEOWNERS into this file & then re-add any changes that need to be part of this branch (if any)?

Reply:
I'm sorry I removed that from the commit I tought it was added by me really didn't know that it was part of oppia android will do all this